beta-src/src/components/ui/WDCommandButtons.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { useState } from "react";
+import { Stack } from "@mui/material";
+import WDCheckmarkIcon from "../svgr-components/WDCheckmarkIcon";
+import WDButton from "./WDButton";
+
+/**
+ * Define the vote props passed to the function.
+ */
+interface voteProps {
+  voteDraw: boolean;
+  votePause: boolean;
+  voteCancel: boolean;
+}
+
+const WDCommandButtons: React.FC<voteProps> = function ({
+  voteCancel,
+  voteDraw,
+  votePause,
+}) {
+  const [draw, setDraw] = useState(voteDraw);
+  const [pause, setPause] = useState(votePause);
+  const [cancel, setCancel] = useState(voteCancel);
```

Comment:
i think the parent will need to have this state because the parent will use the same logic to send the votes to the table. also, we should avoid using boolean props whenever possible. check our dev chat on skynet for a link to why you should avoid them